test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -13453,18 +13453,17 @@ def create_vnode_def(self, name, attrs={}, numnodes=1, sharednode=True,
         del attribs
         return "\n".join(vdef)
 
-    def add_checkpoint_script(self, dirname=None, body=None, mode=None,
-                              runas=None, sudo=False, uid=None, gid=None):
+    def add_checkpoint_abort_script(self, dirname=None, body=None,
+                                    mode=None, abort_time=30):
```

Comment:
Passing mode=None causes chmod() to fail.  We know what will work for a checkpoint_abort script.  For security reasons, I'd suggest we use 0o700 instead of 0o755.

Reply:
I think, it is good to remove  mode as a argument. we can direct use mode=0o700 in chmod().